modules/retail/pages/retail-uyuni-server-setup.adoc

```diff
@@ -0,0 +1,292 @@
++
+Click btn:[Next] to continue.
+
+. Click btn:[Yes] to begin the setup process.
+
+. When setup is complete, click btn:[Next] to continue.
+Take note of the address to access the {productname} {webui}.
+
+. Click btn:[Finish] to complete {productname} setup.
+
+
+
+== Create the Main Administration Account
+
+This section covers how to create your organization's main administration account for {productname}.
+
+[WARNING]
+====
+The main administration account has the highest authority within {productname}.
+Ensure you keep access information for this account secure.
+We recommend that you create lower level administration accounts for organizations and groups.
+Do not share the main administration access details.
+====
+
+
+.Procedure: Setting Up the Main Administration Account
+
+. In your web browser, enter the address for the {productname} {webui}.
+This address was provided after you completed setup.
+For more information, see xref:retail:retail-uyuni-server-setup.adoc#retail-server-setup-yast[].
+
+. Sign in to the {webui}, navigate to the menu:Create Organization[Organization Name] field, and enter your organization name.
+
+. In the menu:Create Organization[Desired Login] and menu:Create Organization[Desired Password] fields, enter your username and password.
+
+. Complete the ``Account Information`` fields, including an email for system notifications.
+
+. Click btn:[Create Organization] to finish creating your administration account.
+
+When you have completed the {productname} {webui} setup, you are taken to the menu:Home[Overview] page.
+
+
+
+== Add Software Channels
+
+Before you register {productname} branch servers and terminals to your {productname} Server, check that you have the {opensuse} product enabled, and the required channels are fully synchronized.
+
+The products you need for this procedure are:
+
+[[opensuse-channels-cli]]
+[cols="1,1,1,1,1", options="header"]
+.OpenSUSE Channels - CLI
+|===
+
+| OS Version
+| Base Channel
+| Client Channel
+| Updates Channel
+| Other Channels
+
+| {opensuse} Leap 15.2
+| opensuse_leap15_2
+| opensuse_leap15_2-uyuni-client
+| opensuse_leap15_2-updates
+| uyuni-proxy-stable-leap-152
+
+|===
+
+include::snippets/add_channels_cli.adoc[]
+
+
+
+== Check Synchronization Status
+
+include::snippets/check_sync_webui_uyuni.adoc[]
+
+include::snippets/check_sync_cli.adoc[]
+
+[IMPORTANT]
+====
+{opensuse} channels can be very large.
+Synchronization can sometimes take several hours.
+====
+
+
+
+== Trust GPG Keys on Clients
+
+include::snippets/trust_gpg.adoc[]
+
+
+
+
+
+// FIXME Starting from here, everything is preliminary
+// Feedback provided by Lukas
+== Create Activation Key for a Proxy (Branch Server) and the Retail Terminals
+
+For more information about creating activation keys, see xref:client-configuration:activation-keys.adoc[].
+
+The activation key for a proxy (branch server) must contain these child channels:
+
+* openSUSE Leap 15.2 Updates (x86_64)
```

Comment:
Can we conditionalise the version numbers here?

Reply:
I don't know how to do this.  Just deleting the version number?

Reply:
Deleting is simplest, if it will make sense without it. Otherwise, you can use {opensuse-version} which gets updated every time we package.

Reply:
These are the entities that get checked and updated every time (from suma-site.yml):

```
    productname: 'SUSE Manager'
    productnumber: '4.1'
    docversion: '4.1'
    minorversion402: '4.0.2'
    saltversion: 3000
    sles-version: 15
    sp-version: SP2 #use for normal text
    sp-vert: sp2 #use for terminal block
    sp-version-l: sp1
    smrproductnumber: 4.1
    opensuse-version: 15.2
```

Reply:
sure, thanks for the reminder :facepalm:

Reply:
I think we'll also need something for 15_2 and 152.

Reply:
done as good as possible.